src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -705,15 +705,16 @@ nlohmann::json ExportQueryExecutionTrees::computeQueryResultAsQLeverJSON(
   j["runtimeInformation"]["meta"] = nlohmann::ordered_json(
       qet.getRootOperation()->getRuntimeInfoWholeQuery());
   RuntimeInformation runtimeInformation = qet.getRootOperation()->runtimeInfo();
+
+  size_t resultSize = runtimeInformation.numRows_;
+
```

Comment:
Important Note: For lazy results the "total count" with this change will mean absolutely nothing when the outermost operation doesn't support limits. In this case it will just mean how many rows there would've been if the last (and first with offset) block would've been sent too. So in the current code state this will never be higher than `j["res"].size() + 2 * index block size`